tst/utils/NoOPAudioVideoFacade.tsx

```diff
@@ -0,0 +1,16 @@
+import { AudioVideoObserver, DefaultAudioVideoFacade } from "amazon-chime-sdk-js";
+
+export default class NoOPAudioVideoFacade extends DefaultAudioVideoFacade {
+  constructor() {
+    // @ts-ignore
+    super(new NoOpAudioVideoController(), undefined, undefined, undefined, undefined, undefined);
+  }
+
```

Comment:
Will it be easier to just have a sendMetrics API to trigger manually metric event where you can just send a FakeClientMetricReports that extends `DefaultClientMetricReport` and just set all the metrics without the need to import all the index frame from js sdk?

Reply:
Got the error:
> Class constructor DefaultAudioVideoFacade cannot be invoked without 'new'

when trying to:
```js
  class FakeClientMetricReport extends DefaultClientMetricReport {
    constructor() {
      super(new NoOpDebugLogger());
    }
    ...
  }

  const clientMetricReport = new FakeClientMetricReport();
```

Changing the `compilerOptions.target` from `es5` to `es6` in `tsconfig.json` fixes this error but I don't think we want to do this. Any idea on how to fix this issue?

Reply:
<img width="697" alt="image" src="https://user-images.githubusercontent.com/20471438/158647794-018cc732-62fc-45f8-89f2-61098bf8ad25.png">

Reply:
Upgraded the `compilerOptions.target` from es5 to es6 in `tsconfig.json`. The issue is addressed.